### PR TITLE
Rename bg-prov to cbnt-prov

### PR DESCRIPTION
**Pull request by ChriMarMe**
#210 

Signed-off-by: Christopher Meis <christopher.meis@9elements.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,9 @@
 /txt-suite
 /txt-prov
 /bg-prov
```
Comment (walterchris) [inline .gitignore:3]: Does that need to stay here?

File: .gitignore [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
+/cbnt-prov
 cmd/txt-prov/txt-prov
 cmd/bg-prov/bg-prov
```
Comment (walterchris) [inline .gitignore:6]: Same here

File: cmd/cbnt-prov/cmd.go [hunk 3]
```diff
@@ -382,19 +382,19 @@ func (g *generateKMCmd) Run(ctx *context) error {
...
-		options = bgo
+		options = cbnto
 	} else {
-		var bgo bg.BootGuardOptions
-		bgo.BootPolicyManifest.BPMH.BPMRevision = g.Revision
-		bgo.BootPolicyManifest.BPMH.BPMSVN = g.SVN
-		bgo.BootPolicyManifest.BPMH.ACMSVNAuth = g.ACMSVN
-		bgo.BootPolicyManifest.BPMH.NEMDataStack = g.NEMS
+		var cbnto cbnt.BootGuardOptions
+		cbnto.BootPolicyManifest.BPMH.BPMRevision = g.Revision
```
Comment (zaolin) [inline cmd/cbnt-prov/cmd.go:394]: cbnto?

Comment (ChriMarMe) [reply cmd/cbnt-prov/cmd.go:394]: Ah, thanks. Should rename BootGuardOptions as well.

Comment (ChriMarMe) [reply cmd/cbnt-prov/cmd.go:394]: Renamed it to Options. Access now via cbnt.Options

**Review by PatrickRudolph — APPROVED** [review]

**Review by walterchris — APPROVED** [review]

File: cmd/cbnt-prov/cmd.go [hunk 4]
```diff
@@ -307,23 +307,23 @@ func (bpme *bpmExportCmd) Run(ctx *context) error {
...
-		bgo, err := bg.ParseConfig(g.Config)
+		cbnto, err := cbnt.ParseConfig(g.Config)
 		if err != nil {
 			return err
 		}
-		options = bgo
+		options = cbnto
 	} else {
-		var bgo bg.BootGuardOptions
+		var cbnto cbnt.BootGuardOptions
```
[the remark this answers is not in the document]
Comment (ChriMarMe) [reply cmd/cbnt-prov/cmd.go]: This is already outdated. BootGuardOptions is called Options already

**Pull request merged**